src/migrations/20230106142545-fix-onchain-send-usd.ts

```diff
@@ -0,0 +1,28 @@
+module.exports = {
```

Comment:
did you test it locally?

Reply:
Yup, both on old-code-version and new-code-version txns